src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -199,8 +191,30 @@ public void start(Stage primaryStage) throws Exception {
         thestage.show();
     }
 
-    private void buttonClicked(ActionEvent e)
-    {
+    private void buttonClicked(ActionEvent e) {
```

Comment:
Now that you have multiple buttons in this class, it's probably a good idea to rename this function. You probably want to name it based on what it does instead of when it's called.